pkg/kubelet/kubelet.go

```diff
@@ -424,6 +424,12 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		LowThresholdPercent:  int(kubeCfg.ImageGCLowThresholdPercent),
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.ImageMaximumGCAge) {
+		imageGCPolicy.MaxAge = kubeCfg.ImageMaximumGCAge.Duration
+	} else if kubeCfg.ImageMaximumGCAge.Duration != 0 {
```

Comment:
I would probably check for nil in addtion to zero.  

`ImageMaximumGCAge` is omitEmpty so I think it could be empty.

Reply:
Hmm... So the staging api says this is omity empty but the kubelet/apis/config/type.go is not..

Not sure if this would actually be nil..

If Kubelet was upgraded and restarted, I assume it reads the config first and defaults this to zero?

Reply:
> I would probably check for nil in addtion to zero.

atually ImageMaximumGCAge is a [struct](https://github.com/kubernetes/kubernetes/pull/121275/files#diff-8a6475c76df7985730ad410e5d1a6caccbbe3ba44853ed773722063cf3625639R200) so a nil check fails

> Hmm... So the staging api says this is omity empty but the kubelet/apis/config/type.go is not..

I followed ImageMinimumGCAge here, not sure why it's omit empty on one but not the other